README.md

```diff
@@ -194,6 +207,16 @@ So for example, if we had `<dist>` `go-ipfs` and `fs-repo-migrations`, we might
 
 We call this the **distribution index**, the listing of all distributions, their versions, and platform assets.
 
+### Notes on reproducible builds
+
+Running `./dockerized make publish` will produce binaries using the same
+runtime as CI. The main difference between local build and official CI one is
+signing step on platforms such as `darwin` (macOS).
+
+Signatures are attached at the end of macOS binaries, which means
+`*_darwin-*.tar.gz` produced by CI will have additional bytes when compared
+with local build.
```

Comment:
TODO (post PR): Let's add some references to tools for how to do the binary stripping so people don't have to search for it.

Reply:
Filled https://github.com/ipfs/distributions/issues/377 so we dont forget ;)